docs/audit-log-filter/audit-log-filter-privileges.md

```diff
@@ -0,0 +1,7 @@
+# Audit Log Filter privileges
+
+Define the privilege at runtime at the startup of the server. The associated Audit Log Filter privilege can be unavailable if the plugin is not enabled.
+
+## `AUDIT_ADMIN`
+
+This privilege is defined by the [`audit_log_filter` plugin](/audit-log-filter-overview.md) and enables the user to configure the audit log filter plugin.
```

Comment:
[`audit_log_filter` plugin](/audit-log-filter-overview.md) -> the link does not work

Reply:
Redone